docs/src/main/sphinx/admin/preview-web-interface.md

```diff
@@ -30,3 +30,46 @@ Once activated, users can access the interface in the URL context `/ui/preview`
 after successful login to the [](/admin/web-interface). For example, the full
 URL on a locally running Trino installation or Trino docker container without
 TLS configuration is [http://localhost:8080/ui/preview](http://localhost:8080/ui/preview).
+
+## Authentication
+
+The Preview Web UI requires users to authenticate. If Trino is not configured to
+require authentication, then any username can be used, and no password is
+required or allowed. The UI shows the login dialog for password authentication
+with the password input deactivated. This is also automatically the case if the
```

Comment:
@wendigo @koszti

Reply:
yep it looks good to me, it's clear now. thanks